Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10-alpine
+FROM python:3.10
```

Comment:
Switched to the full version of the python docker image because there was some incompatibility with the alpine version and redis on M1 macs